### PR TITLE
add wallclock time

### PR DESCRIPTION
**Pull request by TyOverby**

### CODE REVIEW

**Comment by TyOverby** [comment]
@dotnet/roslyn-infrastructure 
@dotnet/roslyn-perf 

I added some helpful console logs for tracking wallclock time when we don't have CPC available.  I used this recently to run tests on my machine.

File: src/Test/Perf/Utilities/TraceManager.cs [hunk 1]
```diff
@@ -131,8 +138,17 @@ public void Initialize()
         {
         }
 
+        // We have one WallClockTraceManager per test, so we don't
+        // need to worry about other tests showing up
         public void Cleanup()
         {
+            var totalDuration = durations.Aggregate(0, (t, v) => t + v);
```
Comment (jasonmalinowski) [inline src/Test/Perf/Utilities/TraceManager.cs]: There is .Sum()...

File: src/Test/Perf/Utilities/TraceManager.cs [hunk 2]
```diff
@@ -141,6 +157,7 @@ public void EndEvent()
 
         public void EndScenario()
         {
+            durations.Add(System.Environment.TickCount - currentStartTime);
```
Comment (jasonmalinowski) [inline src/Test/Perf/Utilities/TraceManager.cs]: What happens when TickCount restarts?

File: src/Test/Perf/Utilities/TraceManager.cs (hunk 1 above)
Comment (TyOverby) [inline src/Test/Perf/Utilities/TraceManager.cs]: Done

File: src/Test/Perf/Utilities/TraceManager.cs (hunk 2 above)
Comment (TyOverby) [inline src/Test/Perf/Utilities/TraceManager.cs]: Hmm, maybe I should use stopwatch

Comment (TyOverby) [inline src/Test/Perf/Utilities/TraceManager.cs]: I use stopwatch now

File: src/Test/Perf/Utilities/TraceManager.cs [hunk 3]
```diff
@@ -119,9 +122,14 @@ public void ResetScenarioGenerator()
         }
     }
 
-    public class NoOpTraceManager : ITraceManager
+    public class WallClockTraceManager : ITraceManager
     {
-        public NoOpTraceManager()
+        private List<long> durations = new List<long>();
+        private string testName = "";
```
Comment (jaredpar) [inline src/Test/Perf/Utilities/TraceManager.cs]: Private fields need to be `_` prefixed.

File: src/Test/Perf/Utilities/TraceManager.cs [hunk 4]
```diff
@@ -119,9 +122,14 @@ public void ResetScenarioGenerator()
...
+
+        private Stopwatch stopwatch;
```
Comment (jaredpar) [inline src/Test/Perf/Utilities/TraceManager.cs]: Same for all private fields.

File: src/Test/Perf/Utilities/TraceManager.cs [hunk 5]
```diff
@@ -119,9 +122,14 @@ public void ResetScenarioGenerator()
         }
     }
 
-    public class NoOpTraceManager : ITraceManager
+    public class WallClockTraceManager : ITraceManager
     {
-        public NoOpTraceManager()
+        private List<long> durations = new List<long>();
```
Comment (jaredpar) [inline src/Test/Perf/Utilities/TraceManager.cs]: Consider making readonly

Comment (TyOverby) [inline src/Test/Perf/Utilities/TraceManager.cs]: Done

File: src/Test/Perf/Utilities/TraceManager.cs (hunk 4 above)
Comment (TyOverby) [inline src/Test/Perf/Utilities/TraceManager.cs]: Done

**Pull request merged**